### PR TITLE
Remove publisher wins.

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue #2285.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by kostyalmm** [comment]
@tofumatt I am not quite sure what else to do with this PR 🤔 
Seems that all the tasks mentioned in the IB are already done. I removed the story as it was remaining. We can't remove the CSS yet as it is used in other win notifications.

What else would you like me to do in this?

**Comment by tofumatt** [comment]
@kostyalmm I think you're right and that's it—the story seems to be the only thing left out of the PR that I can tell from a quick look.

I don't think there's much else to do on this PR, so assigning it to @felixarntz, @aaemnnosttv, or @eugene-manuilov for review is all that's needed. Writing the IB on this one meant finding a lot of the code to remove and ended up warranting the draft PR. I guess there's less left to do here than expected. 🙂

**Comment by eugene-manuilov** [comment]
@googlebot I consent.

**Comment by kostyalmm** [comment]
@tofumatt can you please consent 😄

**Comment by tofumatt** [comment]
@googlebot I consent.

**Comment by tofumatt** [comment]
@kostyalmm I think you need to consent; it's a bit confusing, but I think the idea is the author of the PR is assumed to be okay, but since you added commits to a PR I started you need to consent. 🙂

**Comment by kostyalmm** [comment]
@googlebot I consent.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @tofumatt! Overall it looks good but I think there are a few more things to delete or update:

* [WinsWithData](https://github.com/google/site-kit-wp/blob/publisher-wins-2285/assets/js/components/legacy-notifications/wins-withdata.js) class is not needed anymore
* Comments [here](https://github.com/google/site-kit-wp/blob/publisher-wins-2285/assets/js/components/legacy-notifications/util.js#L38) and [here](https://github.com/google/site-kit-wp/blob/publisher-wins-2285/assets/js/components/data/index.js#L85) need to be updated.

**Review by eugene-manuilov — APPROVED** [review]
LGTM!

File: stories/notifications.stories.js [hunk 1]
```diff
@@ -185,36 +185,6 @@ storiesOf( 'Global/Notifications', module )
...
-						title: 'Average CTR',
-						datapoint: 12.9,
-						datapointUnit: '%',
-					},
-				]
-			}
-			type="win-stats"
-		/>
-	) )
 	.add( 'Total Stats', () => (
```
Comment (felixarntz) [inline stories/notifications.stories.js:188]: There are a few more stories to remove here. Most of the ones below here are also publisher wins - they basically replicate the UI the publisher wins code removed would have resulted in. (Probably the production logic was too complicated to rely on for Storybook - good to get that removed finally. 😆)

**Review by felixarntz — CHANGES_REQUESTED** [review]

**Review by felixarntz — APPROVED** [review]
LGTM! So much red 😍

**Pull request merged**